sphinx/themes/basic/static/searchtools.js

```diff
@@ -293,10 +293,12 @@ const Search = {
       if (title.toLowerCase().trim().includes(queryLower) && (queryLower.length >= title.length/2)) {
         for (const [file, id] of foundTitles) {
           let score = Math.round(100 * queryLower.length / title.length)
+          let isDocumentTitle = titles[file] === title
+          let sectionId = id ? "#" + id : ""
```

Comment:
```suggestion
          let anchor = id ? "#" + id : ""
```

(to correspond to the documentation on [L287](https://github.com/sphinx-doc/sphinx/pull/11942/files#diff-52f537c2146ef4388e77c5e7616aecdabb7fc09454d03694ae238551b9fc1286R287))